extensions/pkg/controller/status_test.go

```diff
@@ -19,13 +19,14 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/go-logr/logr"
```

Comment:
Nit, Should be grouped with line 31 group.

Reply:
We agreed a long time ago, that we don't want to enforce manually sorting/grouping the imports.
That's not of any value if we have to do it manually.
We accept everything is accepted by `gofmt/goimports`. 

If we do want to have stricter sorting/grouping, we should introduce a tool to perform and verify it. However, as the upstream community doesn't provide a good tool for that, we decided to stop caring about it.